fvp/bl2_sp_list.dtsi

```diff
@@ -48,4 +48,9 @@ test_sp3 {
 	load-address = <0x7a20000>;
 };
 
+test_sp4 {
+	/* FF-A UUID */
```

Comment:
remove this inline comment? (for consistency within this DTSI file)

Reply:
Ack